src/main/java/javaserver/ResponseBuilders/DirectoryResponseBuilder.java

```diff
@@ -0,0 +1,42 @@
+package javaserver.ResponseBuilders;
+
+import javaserver.RequestHandlers.DirectoryHandler;
+import javaserver.HTMLContent;
+import javaserver.HTTPStatusCode;
+import javaserver.Routes;
+
+public class DirectoryResponseBuilder extends ResponseBuilder {
+	DirectoryHandler requestHandler;
+
+	public DirectoryResponseBuilder(DirectoryHandler directoryRequestHandler) {
```

Comment:
Same comment regarding consistency with the constructor access modifiers. I notice that it's usually omitted, but here you are defining it.